packages/terra-progress-bar/docs/progress-bar-color.md

```diff
@@ -0,0 +1,51 @@
+By passing in a colorClass, authors can manually set the bar color of any `<ProgressBar />` component.
+
+```css
+.color-bar-example-1 {
+  color: rgb(255, 165, 0);
```

Comment:
Two items wholistically for terra-progress-bar:
1. need to add `--my-app-...` themeable variables to all example actual css and doc-info sample css as best practice and theming requirement for proper usage.
2. would be nice to remove extra white-space and line-returns in the sample css to tighten up cognitive "space" between example info/css and example visuals (even though will not match what stylelint linter would expect).

Reply:
Fixed in https://github.com/cerner/terra-core/pull/2211/commits/618ae6178edc4a734ef31a044c3bc8d570051b5c